teps/NNNN-tep-template/README.md

```diff
@@ -238,6 +238,12 @@ All code is expected to have adequate tests (eventually with coverage
 expectations).
 -->
 
+# Design Evaluation
+<!--
+How does this proposal affect the reusability, simplicity, flexibility 
+and conformance of Tekton, as described in [design principles](../../design-principles.md)
```

Comment:
total nitpick but you might want to put the full link e.g. https://github.com/tektoncd/community/blob/master/design-principles.md cuz i dont think the markdown link in the comment will get rendered

Reply:
updated the link